### PR TITLE
Keep response codes and revert metadata

### PR DESCRIPTION
**Pull request by sandhilt**
Based on PR: https://github.com/cartesi/openapi-interfaces/pull/16

### CODE REVIEW

**Comment by guidanoli** [comment]
The `prevRandao` field has 32 bytes, so typing it as `uint64` can be unsafe.

**Comment by Bankpenk** [comment]
PC serial no : 3E184361H
Challenge code : B2WE8-Y9QTU-HLBYN-YF9WG-QYXN3
Response :

Plis help me

**Comment by guidanoli** [comment]
Before the Merge, the `0x44` opcode (then called `DIFFICULTY`) used to push the current block difficulty onto the EVM stack, which could indeed fit into a `uint64`. However, post-Merge, this opcode was renamed as `PREVRANDAO` and now pushes the latest `mixHash` value, which is totally random, and will, most certainly overflow a `uint64`. See [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399) for reference.

**Comment by sandhilt** [comment]
Hello @guidanoli. Thank you for you feedback. I change `prev_randao` see here:
https://github.com/cartesi/openapi-interfaces/pull/17/commits/05965df6d99e331bf8fa9316da2aeb444a77fb41

Feel free If need more changes.

File: rollup.yaml [hunk 1]
```diff
@@ -362,8 +362,9 @@ components:
           description: Unix timestamp of block in milliseconds.
           example: 1588598533000
         prev_randao:
-          type: integer
-          format: uint64
+          type: string
+          format: hex
+          pattern: "^0x([0-9a-fA-F]{2})*$"
```
Comment (guidanoli) [inline rollup.yaml:367]: I would be more strict and say this is `^0x([0-9a-fA-F]{64})$`

Comment (sandhilt) [reply rollup.yaml]: @guidanoli Updated!

**Pull request closed without merging**